sdk/compute/Azure.Management.Compute/README.md

```diff
@@ -0,0 +1,69 @@
+# README.md template
```

Comment:
These README's are still in a bogus state, please file an issue to get them fixed before we end up shipping them this way.